src/main/java/com/eerussianguy/firmalife/init/AgingFL.java

```diff
@@ -0,0 +1,41 @@
+package com.eerussianguy.firmalife.init;
+
+import net.minecraft.util.IStringSerializable;
+
+import mcp.MethodsReturnNonnullByDefault;
+import net.dries007.tfc.api.capability.food.FoodTrait;
+
+@MethodsReturnNonnullByDefault
+public enum AgingFL implements IStringSerializable
```

Comment:
I don't really care but this is better as an inner class to the cheese wheel.

Reply:
I implemented it as a separate Enum in case we'd want to use it for other aged foods, for now it could be internal but i'll leave as is.